miri-script/src/commands.rs

```diff
@@ -159,14 +160,14 @@ impl Command {
     pub fn exec(self) -> Result<()> {
         // First, and crucially only once, run the auto-actions -- but not for all commands.
         match &self {
-            Command::Install { .. }
-            | Command::Build { .. }
-            | Command::Check { .. }
-            | Command::Test { .. }
-            | Command::Run { .. }
-            | Command::Fmt { .. }
-            | Command::Doc { .. }
-            | Command::Clippy { .. } => Self::auto_actions()?,
+            Command::Install { features, .. }
+            | Command::Build { features, .. }
+            | Command::Check { features, .. }
+            | Command::Test { features, .. }
+            | Command::Run { features, .. }
+            | Command::Doc { features, .. }
+            | Command::Clippy { features, .. } => Self::auto_actions(features.clone())?,
+            | Command::Fmt { .. } => Self::auto_actions(vec![])?,
```

Comment:
`Fmt` does not have features, however the features might be needed for some auto action. Should I add `features` to `Fmt` too, and expand its doc to explain how it's not actually used for Fmt but for any auto action?

Reply:
auto-clippy should be indistinguishable from `./miri clippy`, so no features, yes.

Reply:
Ok, I removed `features` from auto-actions altogether, thanks for the input